Spelling, grammar, and link check

trevorgerhardt opened the pull request:
I've been using Codex recently and asked it to go through our docs and check for any spelling, grammar, or link issues.

trevorgerhardt committed:
Spellchecking, link fixes, minor improvement

trevorgerhardt committed:
More spelling fixes and grammar improvements

ansoncfit commented on docs/analysis/index.mdx:
```diff
@@ -53,7 +53,11 @@ To change the origin of the analysis, drag the marker to a new location.
+## Accessibility chart
+
+When an opportunity layer is selected, the histogram above the map shows how many destinations are reachable at the current travel time cutoff and percentile. Adjust the sliders to explore different thresholds and watch the chart update instantly. Hovering over the chart highlights the percentile and displays the exact count of reachable opportunities at that level.
+
```
Not accurate -- we don't have a histogram above the map or exact counts displayed on hover.

```suggestion
```

ansoncfit commented on docs/analysis/index.mdx:
```diff
@@ -53,7 +53,11 @@ To change the origin of the analysis, drag the marker to a new location.
 
-The display of modifications on the map is controlled in editing mode (See: [Toggling display of modifications](/edit-scenario/usage#toggle-mod-display)). To change what modifications are shown, you must set the visibility there and then navigate back to the analysis page. 
+## Accessibility chart
+
```
```suggestion
```

ansoncfit commented on docs/analysis/index.mdx:
```diff
@@ -24,7 +24,7 @@ The main analysis page is for generating [isochrones](/glossary#isochrone) (trav
-If this is your first time performing an analysis with a given [network bundle](/glossary#network-bundle) it may take some time to build the network, however this only needs to be done once for each bundle. For more information, see the [FAQ page](/learn-more/faq#why-the-wait).
+If this is your first time performing an analysis with a given [network bundle](/glossary#network-bundle) it may take some time to build the network, however this only needs to be done once for each bundle. For more information, see the [FAQ page](/learn-more/faq#how-long-should-it-take-to-start-a-single-origin-analysis).
 
 Once the compute cluster is initialized you should see an isochrone displayed in blue around your point on the map. If you have selected an destination opportunity layer, you will also see a [chart of accessibility](#accessibility-chart) at selected time and percentile thresholds. You may also select a comparison project and scenario, which will be shown in red.
```
```suggestion
Once the compute cluster is initialized you should see an isochrone displayed in blue around your point on the map. If you have selected an destination opportunity layer, you will also see a [chart of accessibility](#chart-of-accessibility-results) at selected time and percentile thresholds. You may also select a comparison project and scenario, which will be shown in red.
```

ansoncfit commented on docs/analysis/configuration.mdx:
```diff
@@ -24,7 +24,7 @@ There may be variability in travel times, depending on when exactly in a specifi
 ## Bookmarks
 
-The first panel allows the creation and use of **bookmarks**, which store particular analysis settings (e.g. origin location, type of opportunity, departure date and time, travel time cutoff, etc.). Once you have the settings the way you want to save them, you can click the grean *Create* button to create a new bookmark. Once created, you can select a bookmark from the dropdown box to automatical ...
+The first panel allows the creation and use of **bookmarks**, which store particular analysis settings (e.g. origin location, type of opportunity, departure date and time, travel time cutoff, etc.). Once you have the settings the way you want to save them, you can click the green *Create* button to create a new bookmark. Once created, you can select a bookmark from the dropdown box to automatical ...
```
✅

ansoncfit commented on docs/prepare-data/derivations.mdx:
```diff
@@ -13,7 +13,7 @@ To use certain features, you first need to prepare Conveyal-specific data by der
 
 ## Aggregation Areas
-Aggregation areas are used to [summarize regional analysis results](/analysis/regional#measuring-aggregate-accessibility). To create an aggregation area, use the [Data Sources](/prepare-data) panel to upload and process a polygon shapefile. Options to create aggregation areas from other spatial data source formats (e.g. geopackage and GeoJSON files) are coming soon.
+Aggregation areas are used to [summarize regional analysis results](/analysis/regional#measuring-aggregate-accessibility). To create an aggregation area, use the [Data Sources](/prepare-data) panel to upload and process a polygon shapefile or polygon GeoJSON. Support for additional spatial formats (e.g. geopackage files) may be added in future releases.
```
For now, aggregation areas can only be created from shapefiles.

```suggestion
Aggregation areas are used to [summarize regional analysis results](/analysis/regional#measuring-aggregate-accessibility). To create an aggregation area, use the [Data Sources](/prepare-data) panel to upload and process a polygon shapefile. Options to create aggregation areas from other spatial data source formats (e.g. geopackage and GeoJSON files) may be added in future releases.
```

ansoncfit commented on docs/edit-scenario/usage.mdx:
```diff
@@ -16,12 +16,14 @@ import {
 import Image from '../../components/image'
 
+## Managing modifications {#managing-modifications}
+
```
Could this approach be used for other links?

trevorgerhardt replied:
It could, but seems a bit redundant. Not sure why Codex went with that approach here.

I enabled config settings that should throw errors for broken links in the future.

I think I'll remove this `{#}` ref to be consistent.

ansoncfit commented on docs/analysis/regional.mdx:
```diff
@@ -55,13 +55,15 @@ Select at least one Destination opportunity layer:
 * When using freeform spatial datasets as destinations in a regional analysis, you can specify only a single layer for now. The maximum number of freeform destination points is 4 million (and the maximum number of freeform origin-destination pairs is 16 million). Grid and freeform destinations cannot yet be mixed.
 
+If you only need results for matched origin-destination pairs (rather than every combination), activate **Linked origin-destination pairs**. This limits CSV output to the paired points in your uploaded freeform datasets and can substantially reduce download sizes. For more detail on configuring paired results, see the [custom points guide](/guides/compute-freeform#linked-origin-destination-pairs) ...
+
```
```suggestion
If you only need results for matched origin-destination pairs (rather than every combination), activate **Linked origin-destination pairs** (more details [here](/guides/compute-freeform#linked-origin-destination-pairs)).

```

ansoncfit commented on docs/analysis/regional.mdx:
```diff
@@ -42,7 +42,7 @@ You can use a regional analysis to compute different types of results. Options u
 * Times: [travel time matrix](/guides/compute-freeform#travel-time-matrix) for each origin-destination pair
 * Paths: details of [public transport itineraries](/guides/compute-freeform#path-details) for each origin-destination pair
-* Dual access: time needed to reach a threshold number of destinations. For more details, see [this paper](https://doi.org/10.1111/gean.12220) or the [TransitCenter Equity Dashboard](https://dashboard.transitcenter.org). If this option is selected, you will also see a field to enter the desired thresholds. For example, 3 would yield time needed to reach third-nearest grocery store with grocery st ...
+* Dual access: time needed to reach a threshold number of destinations. For more details, see [this paper](https://doi.org/10.1111/gean.12220) or the [TransitCenter Equity Dashboard](https://dashboard.transitcenter.org). If this option is selected, you will also see a field to enter the desired thresholds. For example, 3 would yield time needed to reach third-nearest grocery store with grocery st ...
```
It might make more sense to specify that Times and Paths results are only available when both the origin and destination layers are freeform (non-grid) points.

trevorgerhardt replied:
Updated this section to reflect your comment inline and separate out the "Dual access" paragraph into a subsection because it is much longer than the other bullet points.

ansoncfit reviewed (changes requested):
The spelling and typo fixes are worth making, thanks.

Regarding broken links, many of the section heading targets were removed in https://github.com/conveyal/docs/commit/69365b31a7d492df94f8aa1e8284f4401aea1856. If we can use aliases in MDX, the shorter previous tags `{#why-the-wait}`, `{#how-long}`, and `{#access-chart}` might be preferable. But I'm not adamant about that, and it may be easier for long-term maintenance to just use the full section headings.

Some of the generated text was incorrect. It would have been easier to review the trivial spelling changes if other changes that required verification/correction were separate.

trevorgerhardt committed:
Apply suggestion from @ansoncfit

Co-authored-by: Anson Stewart <astewart@conveyal.com>

trevorgerhardt committed:
Apply suggestion from @ansoncfit

Co-authored-by: Anson Stewart <astewart@conveyal.com>

trevorgerhardt committed:
Apply suggestion from @ansoncfit

Co-authored-by: Anson Stewart <astewart@conveyal.com>

trevorgerhardt committed:
Apply suggestion from @ansoncfit

Co-authored-by: Anson Stewart <astewart@conveyal.com>

trevorgerhardt committed:
Apply suggestion from @ansoncfit

Co-authored-by: Anson Stewart <astewart@conveyal.com>

trevorgerhardt committed:
Enable `onBrokenMarkdownLinks` and `onBrokenAnchors`

trevorgerhardt committed:
Remove redundant anchor

trevorgerhardt committed:
Re-arrange Regional Result Types section

- Add notes that Times and Paths require frreeform origins and destinations
- Separate out dual access results details into its own section and add a link to its freeform section.

trevorgerhardt commented:
> Regarding broken links

Using section headings themselves (if we can) is better IMO. Less redundancy. I also added a configuration option to check for and throw errors if links are broken.

> It would have been easier to review the trivial spelling changes if other changes that required verification/correction were separate.

Yes, I agree. I had intended to split them out fully but in my haste included some more verbose changes.

ansoncfit reviewed (approved):
Thanks for incorporating the feedback.
